core/shared/src/main/scala/cats/effect/Bracket.scala

```diff
@@ -57,7 +59,8 @@ trait Bracket[F[_], E] extends MonadError[F, E] {
    * @param use $useParam
    * @param release is the action that's supposed to release the
    *        allocated resource after `use` is done, irregardless of
-   *        its exit condition
+   *        its exit condition. Throwing inside this function
+   *        is an undefined behaviour since it's left to the implementation.
```

Comment:
```suggestion
   *        is undefined behavior since it's left to the implementation.
```

Consistent with above suggestion.